CHANGELOG.md

```diff
@@ -3,6 +3,42 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.10.0
+
+_Apr 25, 2022_
+
+We'd like to offer a big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Introduce [Row reorder](https://mui.com/x/react-data-grid/rows/#row-reorder) (#4034) @DanailH
+
+  <img src="https://user-images.githubusercontent.com/5858539/165073725-b1cbfc62-3935-4bf8-8458-f1c30f9b20f8.gif" width="600">
```

Comment:
@DanailH could you record the gif in a bigger resolution? All other gifs were at least 800px of width.

Reply:
Updated